README.md

```diff
@@ -13,10 +13,62 @@ Requires Blizzard API files (see below).
 
 ------
 
-CommandCenter bots are supported.  Just add the json config file in the command center directory with the .ccbot extension.
 
-LadderManager will read a config file from the LadderManager.conf file with the following values
+# Developer Install / Compile Instructions
+## Requirements
+* [CMake](https://cmake.org/download/)
+* [s2client-api](https://github.com/Blizzard/s2client-api/tree/master) 
```

Comment:
You don't need s2client-api as it is submodule now.

Reply:
Removed